client/src/Questions/QuestionForm.js

```diff
@@ -21,9 +24,19 @@ function QuestionForm() {
 
   useEffect(
     function () {
-      if (id) {
-        Api.questions.get(id).then((response) => setQuestion(response.data));
-      }
+      Api.questionnaires
```

Comment:
In the useEffect, the first things we do is load the list of questionnaires... then we'll check for and load a Question to edit, if we're editing/updating...